Rakefile

```diff
@@ -111,7 +111,7 @@ namespace :build do
       env_vars = ''
       env_vars += 'MOOC_LOCALIZE=1 ' if CDO.localize_apps
       env_vars += 'MOOC_DIGEST=1 ' unless rack_env?(:development)
-      RakeUtils.system "#{env_vars} grunt"
+      RakeUtils.system "#{env_vars} grunt build"
```

Comment:
It is less efficient.  I think the issue generally reported is that our SCSS step and maybe some other steps outside of browserify don't accurately detect new changes so sometimes SCSS doesn't get rebuilt unless you do a clean first.